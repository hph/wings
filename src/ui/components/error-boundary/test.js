import React from 'react';
import renderer from 'react-test-renderer';

import ErrorBoundary from './index';

describe('ErrorBoundary', () => {
  const defaultProps = {
    info: 'Some info about where the error originated',
    message: 'The error message',
    type: 'Error type',
  };

  it('should render an error message with the provided props', () => {
    expect(
      renderer.create(<ErrorBoundary {...defaultProps} />).toJSON(),
    ).toMatchSnapshot();
  });
});
