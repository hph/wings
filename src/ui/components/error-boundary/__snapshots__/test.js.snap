// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should render an error message with the provided props 1`] = `
<div
  className="root"
>
  <div
    className="type"
  >
    Error type
  </div>
  <div>
    The error message
  </div>
  <div
    className="info"
  >
    Some info about where the error originated
  </div>
</div>
`;
